dpericich/Health-Checklist
# [feature] Scaffold Health Checklist App

## Description:
This branch is scaffolding to unblock the rest of the work on this app. It establishes the two pages for our MVP: home input form and checklist results. It also sets up routing, establishes stylesheets and scaffolds dropdown category elements

## Testing:
- pull the latest version of this branch
- start your server and navigate to the home page `localhost:3000`
- Verify that there are 2 dropdown inputs
- Verify the `Show Results` button
- Click on the button and verify that you are sent to `localhost:3000/results`
- Verify Existence of 4 category placeholder elements on the page
- Verify the `Update Inputs` button
- Click on the button and verify that you are sent to `localhost:3000/`
